src/intrusive.rs

```diff
@@ -305,7 +577,17 @@ impl<R: ScopedRawMutex + ConstInit> StorageList<R> {
     /// ```
     pub const fn new() -> Self {
         Self {
-            list: Mutex::new_with_raw_mutex(List::new(), R::INIT),
+            inner: Mutex::new_with_raw_mutex(
+                StorageListInner {
+                    list: List::new(),
+                    seq_state: SeqState {
+                        next_seq: 0,
```

Comment:
From previous comment, I think if this is set to 1, it's already a valid value if no full sequence is found